### ISSUE TITLE
Upgrading kubb/cli from 1.6.1 to 1.6.2 presents error about requring @swc/core or @swc/wasm

### ISSUE THREAD

**Issue by jafin** [kubb-project/kubb#393]
### What version of `kubb` is running?

1.6.2

### What platform is your computer?

Windows

### What steps can reproduce the bug?

Upgrade @kubb/cli from 1.6.1 to 1.6.2
Receive error on execution

```text

> kubb generate --config config.ts

✖ swc compiler requires either @swc/core or @swc/wasm to be installed as a dependency.  
See https://typestrong.org/ts-node/docs/transpilers
  Error: swc compiler requires either @swc/core or @swc/wasm to be installed as a dependency.  
See https://typestrong.org/ts-node/docs/t  ranspilers

kubb/cli  1.6.1 
```

Reverting to 1.6.1 resolves the issue.

### Additional information

Diffing the commits for 1.6.2:

In `packages/cli/package.json`, it appears the below dependency was removed (perhaps related)
```json
"dependencies"
"@swc/core": "^1.3.80",
```

**Comment by stijnvanhulle** [comment]
@jafin Oeps, removed it by accident, I will release a new version :)

**Comment by jafin** [comment]
@stijnvanhulle thanks for the prompt fix, 1.6.3 resolved the issue
